requirements.txt

```
@@ -44,129 +42,125 @@ django==4.2.8
     #   djangocms-frontend
     #   djangocms-versioning
     #   easy-thumbnails
-django-appconf==1.0.6
-    # via django-select2
 django-classy-tags==4.1.0
     # via
     #   django-cms
     #   django-sekizai
-django-cms==4.1.0
+django-cms==4.1.2
     # via
     #   -r requirements.in
     #   djangocms-alias
     #   djangocms-attributes-field
     #   djangocms-frontend
-    #   djangocms-text-ckeditor
+    #   djangocms-text
     #   djangocms-versioning
 django-entangled==0.5.4
     # via djangocms-frontend
-django-filer==3.1.1
+django-filer==3.1.4
     # via
     #   -r requirements.in
     #   djangocms-frontend
 django-formtools==2.5.1
     # via django-cms
-django-fsm==2.8.1
+django-fsm==2.8.2
     # via djangocms-versioning
 django-parler==2.3
     # via djangocms-alias
 django-polymorphic==3.1.0
     # via django-filer
 django-sekizai==4.1.0
     # via django-cms
-django-select2==8.1.2
-    # via djangocms-frontend
 django-storage-url==0.12.0
     # via -r requirements.in
-django-storages==1.14.2
+django-storages==1.14.4
     # via django-storage-url
-django-treebeard==4.7
+django-treebeard==4.7.1
     # via django-cms
-djangocms-admin-style==3.2.6
+djangocms-admin-style==3.3.1
     # via
     #   -r requirements.in
     #   django-cms
-djangocms-alias==2.0.0
+djangocms-alias==2.0.1
     # via -r requirements.in
 djangocms-attributes-field==3.0.0
     # via djangocms-frontend
-djangocms-frontend==1.2.2
+djangocms-frontend==1.3.3
     # via -r requirements.in
-djangocms-text-ckeditor==5.1.5
-    # via
-    #   -r requirements.in
-    #   djangocms-frontend
-djangocms-versioning==2.0.0
+djangocms-text==0.2.1
```

```suggestion
djangocms-text==0.2.2
```

This also works with manifest storage.